Localize/lang/strings.json

```
@@ -2327,6 +2348,7 @@
   "kVwJXt": "(UTC+04:00) Abu Dhabi, Muscat",
   "kYivbS": "Download (Alt/Option + click)",
```

Use "select", not "click", for general accessibility support:

```suggestion
  "kYivbS": "Download (Alt or Option + select)",
```